### PR TITLE
circulation: display ILL request in patron profile

### PR DESCRIPTION
**Pull request by zannkukai**
Adds a tab into the patron profile to display the requests related to
the current logged patron.

## How to test?

image

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+              {{ request.status }}
+            </span>
+          </div>
+        </div>
+
+        <div id="request-{{ request.pid }}" class="mt-1 ng-star-inserted d-none">
+          <h6 class="mt-2 mb-2 ng-star-inserted">{{ _('Details') }}</h6>
+          <section class="col">
+            <div class="row">
+              <div class="col-12 label-title"><b>{{ _('Document information') }}</b></div>
```
Comment (Garfield-fr) [inline rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html:61]: `<div class="col-12 label-title"><b>{{ _('Document information') }}</b></div>` change with `<div class="col-12 label-title font-weight-bold">{{ _('Document information') }}</div>`

the same on the other elements

File: rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                    {% if request.document.source.number %}<span> n°.{{ request.document.source.number }}</span>{% endif %}
+                    <span>)</span>
+                  {% endif %}
+                </div>
+              {% endif %}
+              <div class="col-md-2 col-sm-4"><b>{{ _('Pickup location')}}</b></div>
+              <div class="col-md-10 col-sm-8">{{ request.pickup_location.name }}</div>
+              <div class="col-md-2 col-sm-4"><b>{{ _('Is request copy')}}</b></div>
+              <div class="col-md-10 col-sm-8">
+                {% if request.is_copy %}
```
Comment (Garfield-fr) [inline rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html:91]: `<i class="fa fa-{% if request.is_copy %}check text-success{% else %}times text-danger{% endif %}"></i>`

File: rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                {% endif %}
+              </div>
+              {% if request.pages %}
+                <div class="col-md-2 col-sm-4"><b>{{ _('Pages')}}</b></div>
+                <div class="col-md-10 col-sm-8">{{ request.pages }}</div>
+              {% endif %}
+              {% if request.found_in %}
+                <div class="col-md-2 col-sm-4"><b>{{ _('Found in')}}</b></div>
+                <div class="col-md-10 col-sm-8">
+                  <span>({{ request.found_in.source }}) </span>
```
Comment (Garfield-fr) [inline rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html:104]: remove space after `)`

File: rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                <div class="col-md-2 col-sm-4 pl-5"><b>{{ _('Identifier')}}</b></div>
+                <div class="col-md-10 col-sm-8">{{ request.document.identifier }}</div>
+              {% endif %}
+              {% if request.document.source %}
+                <div class="col-md-2 col-sm-4 pl-5"><b>{{ _('Source')}}</b></div>
+                <div class="col-md-10 col-sm-8">
+                  {{ request.document.source.journal_title }}
+                  {% if request.document.source.volume or request.document.source.number %}
+                    <span class="pl-3">(</span>
+                    {% if request.document.source.volume %}<span> Vol.{{ request.document.source.volume }}</span>{% endif %}
```
Comment (Garfield-fr) [inline rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html:80]: Why `<span>...</span>` here ?

File: rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+                    {% if request.document.source.volume and request.document.source.number %} -- {% endif %}
+                    {% if request.document.source.number %}<span> n°.{{ request.document.source.number }}</span>{% endif %}
```
Comment (Garfield-fr) [inline rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html:82]: Why `<span>...</span>` here ?

File: rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html (hunk 1 above)
Comment (zannkukai) [reply rero_ils/modules/patrons/templates/rero_ils/_patron_profile_ill_requests.html:61]: done

**Review by Garfield-fr — APPROVED** [review]

**Review by lauren-d — APPROVED** [review]

**Pull request merged**